components/button-indigo.html

```diff
@@ -0,0 +1,3 @@
+<div class="flex items-center justify-center min-h-screen p-5 bg-gray-100 min-w-screen">
+    <button class="px-4 py-2 text-lg font-medium tracking-wider text-white bg-indigo-700 rounded-md hover:bg-indigo-900 focus:outline-none focus:shadow-outline focus:bg-indigo-900">Click Me!</button>
```

Comment:
The only thing I would suggest changing is what the button says, to make it a more realistic representation of what it could be used for. 🙂

```suggestion
    <button class="px-4 py-2 text-lg font-medium tracking-wider text-white bg-indigo-700 rounded-md hover:bg-indigo-900 focus:outline-none focus:shadow-outline focus:bg-indigo-900">Register</button>
```